src/theme/base/_mediaqueries.scss

```diff
@@ -32,11 +32,17 @@
   }
 */
 
+$mobile-and-smallest: 320px; // iPhone SE & smaller
 $tablet-and-wider: 768px; // Tablet
 $desktop-and-wider: 1024px; // Desktop
 $desktop-large-and-wider: 1440px; // Large desktops
 
 @mixin respond-to($media) {
+  @if $media == mobile {
```

Comment:
This might be confusing, we are building the app following a mobile first approach, adding a `mobile` media query might confusing I think.

What if we do this for all mobiles:
```
    display: flex;
    justify-content: space-evenly;
    padding: 0 2rem;
```
I think it will look OK on larger mobile screens.

Reply:
I think the same

Reply:
yes, I think that would be better.